scripts/g.extension/g.extension.py

```diff
@@ -2055,6 +2055,9 @@ def install_extension_std_platforms(name, source, url, branch):
 
     os.chdir(srcdir)
 
+    # add srcdir to PYTHONPATH
+    os.environ["PYTHONPATH"] = f"{srcdir}:{os.environ['PYTHONPATH']}"
```

Comment:
The following would be more crossplatform, but since it is not relevant on Windows anyway, you can go with :, I guess...
```suggestion
    os.environ["PYTHONPATH"] = f"{srcdir}{os.pathsep}{os.environ['PYTHONPATH']}"
```

Reply:
Sorry, I was not aware this was blocking a merge...

Reply:
Is it?

Reply:
It was blocking. Any unresolved conversation blocks merging. See "unresolve" button below...

Reply:
And now, after clicking unresolved, it should be possible...

Reply:
Yes, indeed! Done.